content/_landing/create_an_account._en.md

```diff
@@ -25,12 +25,13 @@ steps:
 
 
     * **More secure**
-       * Security Key
-        * Authentication application
-        * Federal government employee or military identification (PIV/CAC)
+      * Security key
+      * Authentication application
+      * Federal government employee or military identification (PIV/CAC)
+      * Face or touch unlock
```

Comment:
There wasn't any specific ordering prescribed in the ticket, but might be worth checking if there's a preference where this is inserted in the list.